clients/crystal/src/pools.cr

```diff
@@ -1,5 +1,6 @@
 require "./nodes"
 require "./volumes"
+require "./heal"
```

Comment:
This is not required from pool. See the above comment

Reply:
done